test/safe/push.test.ts

```diff
@@ -245,92 +245,6 @@ describe.skip("GitHub Actions", () => {
 			await expect(processPush(app)(job)).toResolve();
 		});
 
-		// Commenting these out for the moment. DevInfo API runs these
-		// transitions automatially based on the commit message, but we may
```

Comment:
Removing commented code, we don't need these tests, hence we don't have this logic